examples/js-avo-inspector-example/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "avo-inspector": "^0.1.0",
```

Comment:
What does ^ mean here? Does it try to pick latest available version?

Reply:
Yes, this automatically uses latest minor/patch version (major version not incremented automatically). Generally it is not a good idea to use that for packages as this can break the app with unexpected changes